Source/Scene/Primitive.js

```diff
@@ -1643,6 +1657,19 @@ define([
             }
         }
 
+        var offsetIndex = primitive._batchTableAttributeIndices.offset;
+        if (defined(offsetIndex)) {
+            length = primitive._boundingSpheres.length;
+            for (i = 0; i < length; ++i) {
+                boundingSphere = primitive._boundingSpheres[i];
+                var wc = primitive._boundingSphereWC[i];
+
+                //TODO: associate BS with instance.  Maybe be multiple BS for splitlongitude
+                var offset = primitive._batchTable.getBatchedAttribute(i, offsetIndex, new Cartesian3());
```

Comment:
@bagnell I could use some suggestions for what to do here.  Sometimes there are two bounding spheres for one geometry instance when it's split across the IDL.  Indexing into `getBatchedAttribute` with `i` doesn't work in this case.